tools/tokenserver/test_database.py

```diff
@@ -0,0 +1,446 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file was adapted from [this file](https://github.com/mozilla-services/tokenserver/blob/master/tokenserver/tests/assignment/test_sqlnode.py) on the legacy Tokenserver